src/__tests__/integration.test.ts

```diff
@@ -32,6 +32,6 @@ it("It can make a request against the schema", async () => {
       `,
     })
 
-  expect(response.body).toEqual({ data: { artist: { name: "Mr Bank" } } })
+  expect(response.body.data).toEqual({ artist: { name: "Mr Bank" } })
```

Comment:
@orta On a clean setup of the MP repo I was getting the debug extensions included as well, which wouldn’t match this, so I changed the test to only consider the `data` entry.

Reply:
Yeah,  I think that makes sense, this happened with someone else too - so I prefer this ^ implementation idea